test/transpile.jl

```diff
@@ -195,6 +195,39 @@ end
 
 end
 
+@testset "rightangle_or_arbitrary_phase_shift" begin
+
+    list_params=[
+        ( pi/2, Snowflake.Z90),
+        (-pi/2, Snowflake.ZM90),
+        ( pi,   Snowflake.SigmaZ),
+        ( pi/3, Snowflake.PhaseShift)
```

Comment:
We are missing a few cases.

pi/8  should become pi 8
-pi/8 should become pi 8 dag

Also, I just realized we have duplicate gates. The Phase == Z90 and PhaseDag == ZM90. We should remove the duplication.